tools/executors/webpack/build/schema.json

```diff
@@ -0,0 +1,359 @@
+{
+  "cli": "nx",
+  "id": "build",
+  "description": "Serves an app using @nrwl/web:webpack",
```

Comment:
Is this a good way, to store some nx config in our repo? Wouldn't it be troublesome for maintaining

Reply:
you're totally right. tried removing the schema duplicates, and seems like I overlooked that errors are indeed being processed by the third party executors without me dumping all their schema in here, which makes sense, as it's happening on runtime. great catch, thanks!

Reply:
actually i think i might had something cached when this was working, it actually breaks all the builds and serve commands when i remove the schemas. tried referencing the external schemas, but that didn't work, so ended up reverting the attempt to fix this problem. unless there's a proposal how to deal with this properly, i'd leave it as it is for now